rasa/utils/train_utils.py

```diff
@@ -471,7 +473,14 @@ def create_common_callbacks(
     Returns:
         A list of callbacks.
     """
-    import tensorflow as tf
+    try:
+        import tensorflow as tf
+    except ModuleNotFoundError as e:
+        raise MissingDependencyException(
+            f"Tensorflow does not seem to be installed, but it is needed for this."
```

Comment:
Should we also mention [this page](https://www.tensorflow.org/install/errors) ? TensorFlow will always be installed as part of Rasa Open Source - so if the import fails it's usually a TensorFlow problem with the underlying OS than a Rasa Open Source problem.

Reply:
@wochinge I don't see the specific error caught by Sentry in this list - but we could redirect to this GH Tensorflow [thread ](https://github.com/tensorflow/tensorflow/issues/44110) for resolution? In this case do you think it would be a good idea to create `RasaException` subclass called maybe `ImportException` instead of using `MissingDependencyException` since it would indicate the wrong thing as you just stated?